WhoIsThatServer.Storage/Controllers/ITargetElementController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web.Http;
+using WhoIsThatServer.Storage.Models;
+
+namespace WhoIsThatServer.Storage.Controllers
+{
+    public interface ITargetElementController
+    {
+        /// <summary>
+        /// Adds created target element into DB
+        /// </summary>
+        /// <param name="targetElement">object with id, target's and hunter's ids</param>
+        /// <returns>Json</returns>
+        [HttpPost]
```

Comment:
I doubt you need annotations on the interface

Reply:
But then I can use <inheritdoc/> tag, or is that a bad practise?

Reply:
I mean [HttpPost] part

Reply:
Right, my bad, misunderstood